scripts/get-all-test-paths.sh

```diff
@@ -8,12 +8,15 @@ if [[ $1 == "windows" ]]; then
     # dest=( "${array1[@]}" "${array2[@]}" )
     printf '%s\n' "${array2[@]}" | jq -R . | jq -cs .
 else
-    declare -a array1=( "tests/unit/*.py" "tests/integration/*.py" "tests/distributed/*.py" "tests/system/*.py" "tests/docarray/*.py")
-    declare -a array2=( $(ls -d tests/{unit,integration,distributed,system,docarray}/*/ | grep -v '__pycache__' ))
+    # TODO: enable all distributed/daemon unit tests
```

Comment:
I think this file should not be touched here. Merge accident?

Reply:
oh fuck